core/src/utils/utils.ts

```diff
@@ -403,3 +403,14 @@ export function isFocusable(element: HTMLElement): boolean {
 
   return false;
 }
+
+
+/**
+ * Check if the provided element has a slot that corresponds
+ * to the provided slotName.
+ * @param slotName the name of the slot.
+ * @param element the element to look for the slot within.
+ */
+export const hasSlot = (slotName: string, element: HTMLElement) =>
```

Comment:
Is this not the same function that is include in the other PR?

Reply:
Yes, but since non of them are merged I needed to introduce them in all of them. Then I can handle the merge conflicts once one of them are merged

Reply:
Ah ok, one way of doing it for sure 😄